AUTHORS.txt

```diff
@@ -2,7 +2,6 @@ Lukáš Lalinský <lalinsky@gmail.com>
 Robert Kaye <rob@eorbit.net>
 Philipp Wolfer <ph.wolfer@googlemail.com>
 
-Picard icons by Carlin Mangar <openshift@gmail.com>
-
-The icons in this program are from the Tango Desktop Project
-<http://tango.freedesktop.org/>.
+Picard logo by <http://monkeydo.digital>, © 2015 MetaBrainz Foundation
+Some icons in this program are by Carlin Mangar <openshift@gmail.com>, the
```

Comment:
Anyway, AUTHORS.txt doesn't seem, to me, to be meant as a detailed declaration of authorship/copyright, but just meant to give an overview. If you want to know specifics, `git log` or `git blame` would be what you're after.